ios/FluentUI/Other Cells/BooleanCell.swift

```diff
@@ -25,6 +27,12 @@ open class BooleanCell: TableViewCell {
         set { `switch`.isOn = newValue }
     }
 
+    /// Updates the cell accessory view's `isEnabled` property
+    @objc public var isSwitchEnabled: Bool {
+      get { return `switch`.isEnabled }
```

Comment:
should we also take into account of the super class isEnabled? so that when it is dimmed, switch also reflects that

Reply:
I think it's a good point, will update.